CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * Updated `jackson` version to 2.9.9 (was 2.9.8).
 * Updated `beeju` version to 2.0.0 (was 1.2.1).
 * Updated `circus-train-minimal.yml.template` to include the required `housekeeping` configuration for using the default schema with H2. 
+* Updated `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
```

Comment:
This doesn't look right, 14.1.0 was released on Friday, are you sure this is based off the latest contents of master?

Reply:
Good spot, I hadn't pulled! I have rebased it with master